tests/functional/raw-node/test/GCP/object/delete.js

```diff
@@ -83,14 +83,13 @@ describe('GCP: DELETE Object', function testSuite() {
     });
 
     describe('without existing object in bucket', () => {
-        it('should return 404 and NoSuchKey', done => {
```

Comment:
The `GCP raw-node` tests are functional tests for the `gcpClient` making request/handling responses from `Google Cloud`. It does not go through `Cloudserver`, so I think this should stay as it was.

Reply:
Ah. I remember them now. Do we have tests where GCP goes through the AWSClient?

Reply:
There's  [multipleBackend/delete/deleteGcp.js](https://github.com/scality/cloudserver/blob/development/8.0/tests/functional/aws-node-sdk/test/multipleBackend/delete/deleteGcp.js)...if that's what you're asking for.